sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
```

Comment:
It's helpful but not required to test prerequisites before starting something that has them.  Specifically I might recommend something like

```script
if (-Not (Get-Command "openssl" -ErrorAction SilentlyContinue))
{
  throw "openssl is not availabe.  You will need to install it or add it to the PATH to proceed"
}
```